helm-org-ql.el

```diff
@@ -178,21 +178,27 @@ Is transformed into this query:
   "Return Helm source named NAME that searches BUFFERS-FILES with `helm-org-ql'."
   ;; Expansion of `helm-build-sync-source' macro.
   (helm-make-source name 'helm-source-sync
+    :filtered-candidate-transformer (lambda (candidates _)
+				      (let ((window-width (window-width (helm-window))))
+					(mapcar (lambda (marker)
+						  (unless (markerp marker) (user-error "Unexpected candidate for `helm-org-ql-source'"))
```

Comment:
Is it necessary to test the marker here?  Aren't we passing the candidates ourselves, so they are guaranteed to be the proper type?

Reply:
You are right. It was mostly for safety, but should not be necessary.